.github/workflows/crowdin-download.docs.yml

```diff
@@ -54,18 +57,10 @@ jobs:
           # Uncomment below to debug
           # dryrun_action: true
 
-        env:
-          GITHUB_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
-          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_DOCS }}
-          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
-
       # Convert Simplified Chinese to Traditional #
       - name: Convert Chinese
         uses: freecodecamp/crowdin-action@main
         env:
-          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
-          CROWDIN_API_URL: 'https://freecodecamp.crowdin.com/api/v2/'
-          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_CURRICULUM }}
```

Comment:
Not sure if this is needed. The curriculum project id is removed here, but not added to the global variables or whatever, which would conflict with the other variable of the same name I suppose.

Reply:
Good catch, Tom.  It seems like this *should* be using the `CROWDIN_PROJECT_ID_DOCS` id here, but hasn't been.  Also, the traditional chinese docs don't appear on the list:

![image](https://user-images.githubusercontent.com/15801806/153631224-30c5795d-382d-402f-b42d-216feef707f5.png)

Reply:
@moT01 Oops.
So the action is written to require that ENV value for each step, because *most* of the steps need it. The convert-chinese step is one that doesn't, because it doesn't actually talk to Crowdin - which is why this being set to curriculum instead of docs was never an issue.

@ojeytonwilliams I will PR to add traditional chinese today (and german)